molgenis/molgenis-emx2
# fix: when setting isOidcEnabled is false override env variable

Fixes #3619

What are the main changes you did:
- When the database setting isOidcEnabled is set to false it will used this setting and overrides env OIDC properties

how to test:
- Spin up an instance of emx2 with OIDC env variables set
- See OIDC is enabled 
- Log in with admin
- Set isOidcEnabled to false
- See that OIDC login is disabled

todo:
